File: IT/1/Funzioni/Esercizi/C++/esercizio6.cpp
/* file esercizio6.cpp */

#include <iostream>

using std::cout;
using std::endl;
using std::boolalpha;

bool even(int n) {
	return n % 2 == 0 ? true : false;
}

int main() 
{
	int a = 2;
	int b = 3;

	cout << a << " è pari? " << boolalpha << even(a) << endl;
	cout << b << " è pari? " << boolalpha << even(b) << endl;
}

File: IT/1/Funzioni/Esercizi/Java/Esercizio6.java
/* file Esercizio6.java */

public class Esercizio6 {
	static boolean even(int n) {
		return n % 2 == 0 ? true : false;
	}

	public static void main(String [] args) {
		int a = 2;
		int b = 3;

		System.out.println(a + " è pari? " + even(a));
		System.out.println(b + " è pari? " + even(b));
	}
}

